### ISSUE TITLE
Is the project still active?

### ISSUE THREAD

**Issue by pdelfino** [latexforlawyers/latexforlawyers#1]
I think this idea is great!

However, the project seems to be inactive. Are you guys still working on it?

Best,
Pedro

**Comment by ienissei** [comment]
Haven't heard from anyone lately.
What are your goals?

Best,

**Comment by trmd** [comment]
Yeah, long time since I've done anytthing, but would be great to get it going. My programming skills are not really up to the task, however.

Best,
Tormod
—

Tormod Otter Johansen
tormod.otter.johansen@gmail.com
+46 (0) 739 51 54 41

http://tormodotterjohansen.se/

GPG:
B502 C148 9A6E 6DCB 0B63
CBE9 7D9C 6B9F 7A68 9C7B
